examples/magento-graphcms/pages/checkout/payment.tsx

```diff
@@ -121,17 +122,21 @@ function PaymentPage() {
                   ...mollie_methods,
                 }}
               >
-                <PaymentMethodActionCardListForm step={3} />
+                <PaymentMethodActionCardListForm step={4} />
 
                 <CartSummary editable>
                   <CartTotals />
                 </CartSummary>
 
                 <CouponAccordion />
-
-                <CartAgreementsForm step={2} />
-
-                <PaymentMethodPlaceOrder step={4} />
+                <SubscribeToNewsletter
+                  step={3}
+                  label='Subscribe to our newsletter to stay informed about our new products!'
+                  sx={(theme) => ({ paddingTop: theme.spacings.md })}
```

Comment:
Margin top?

Reply:
Dit is een kopie van de styling van het component wat eerst op die plek zat. Dus ging ervan uit dat ik die styling moest overnemen